src/components/MetricsPane/SelectedSiteMetrics.jsx

```diff
@@ -0,0 +1,203 @@
+import PropTypes from 'prop-types'
+
+import {
+  StyledHeader,
+  StyledMetricsSelector,
+  StyledSummaryMetadataContainer,
+} from './MetricsPane.styles'
+import {
+  BiggerIconCalendar,
+  BiggerIconGlobe,
+  BiggerIconPersonCircle,
+  BiggerIconText,
+  BiggerIconTextBoxMultiple,
+  BiggerIconUser,
+  SelectedSiteActionBar,
+  SelectedSiteContentContainer,
+  SelectedSiteMetricsCardContainer,
+  SelectedSiteSiteCardContainer,
+  StyledMapPinContainer,
+  StyledReefContainer,
+  StyledReefItem,
+  StyledReefItemBold,
+  StyledReefRow,
+  StyledSvgContainer,
+  StyledVisibleBackground,
+} from './SelectedSiteMetrics.styles'
+import { ButtonSecondary } from '../generic'
+import { FilterProjectsContext } from '../../context/FilterProjectsContext'
+import { getIsSiteSelected, zoomToSelectedSite } from '../../helperFunctions/selectedSite'
+import { IconClose } from '../../assets/icons'
+import { IconPersonCircle } from '../../assets/dashboardOnlyIcons'
+import { MAIN_MAP_ID } from '../../constants/constants'
+import { useContext, useState } from 'react'
+import { useMap } from 'react-map-gl'
+import coralReefSvg from '../../assets/coral_reef.svg'
+import mapPin from '../../assets/map-pin.png'
+import useResponsive from '../../hooks/useResponsive'
+import ZoomToSiteIcon from '../../assets/zoom_to_selected_sites.svg?react'
+
+export const SelectedSiteMetrics = ({ selectedSampleEvent, view, setSelectedSampleEvent }) => {
+  const {
+    displayedProjects,
+    getURLParams,
+    mermaidUserData,
+    setSelectedMarkerId,
+    updateURLParams,
+    userIsMemberOfProject,
+  } = useContext(FilterProjectsContext)
+  const [metricsView, setMetricsView] = useState('summary')
+  const { isDesktopWidth } = useResponsive()
+  const isMapView = view === 'mapView'
+  const map = useMap()[MAIN_MAP_ID] // the docs for react-map-gl are not clear on the return object for useMap. This is what works in testing. Further, its not a 'ref' its the actual map instance.
+
+  const handleClearSelectedSampleEvent = () => {
+    setSelectedSampleEvent(null)
+    setSelectedMarkerId(null)
+    const queryParams = getURLParams()
+    queryParams.delete('sample_event_id')
+    updateURLParams(queryParams)
+  }
+
+  const sampleEventAdmins = selectedSampleEvent.project_admins.map((admin) => admin.name).join(', ')
+  const sampleEventOrganizations = selectedSampleEvent.tags?.map((tag) => tag.name).join(', ')
+
+  const handleChangeMetricsView = (event) => {
+    setMetricsView(event.target.name)
+  }
+
+  const selectedSiteHeader = (
+    <StyledVisibleBackground>
+      <SelectedSiteSiteCardContainer>
+        <StyledMapPinContainer>
+          <img src={mapPin} alt="map-pin" />
+        </StyledMapPinContainer>
+        <StyledHeader>{selectedSampleEvent.site_name}</StyledHeader>
+      </SelectedSiteSiteCardContainer>
+    </StyledVisibleBackground>
+  )
+  const selectedSiteBody = isDesktopWidth ? (
```

Comment:
`selectedSiteBody` used to be a nested component. The rest of this diff is just moving existing code into new files